upup/pkg/fi/cloudup/awstasks/load_balancer.go

```diff
@@ -749,7 +755,11 @@ func (_ *LoadBalancer) RenderTerraform(t *terraform.TerraformTarget, a, e, chang
 		tf.CrossZoneLoadBalancing = e.CrossZoneLoadBalancing.Enabled
 	}
 
-	tf.Tags = cloud.BuildTags(e.Name)
+	var tags map[string]string = cloud.BuildTags(e.Name)
+	for k, v := range e.Tags {
+		tags[k] = v
+	}
+	tf.Tags = tags
```

Comment:
Nice, mind adding similar tags to cloudformation too? https://github.com/kubernetes/kops/blob/e131f67b3914f6ff3d9a6a2f79ba753112b6af85/upup/pkg/fi/cloudup/awstasks/load_balancer.go#L891

Reply:
As I dont use CF, i'm not sure how to test it.  I'm getting the following with my attempt:
```
 ~/repos/go/bin/kops update cluster --target=cloudformation --out=.
Using cluster from kubectl context: kube.poc.domain

I0325 11:58:49.635364   15845 builder.go:297] error reading hash file "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": unexpected response code "403 Forbidden" for "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz.sha1": <?xml version="1.0" encoding="UTF-8"?>
<Error><Code>AccessDenied</Code><Message>Access Denied</Message><RequestId>F725F452C5A7B3B0</RequestId><HostId>ilTeVSbESSj5HxGbG2Tej5HiTCsXV6/MMJ9JwiAmPWkNTUED6GrExnpY3AYfKrJlNZnW5dgH7mk=</HostId></Error>

cannot determine hash for "https://kubeupv2.s3.amazonaws.com/kops/1.12.0-alpha.1/linux/amd64/utils.tar.gz" (have you specified a valid file location?)
```
 Also with the change I have made, tests appear to be passing, so either there are no current tests which do any sort of tag verification in cf, or my code isnt working at all.

Reply:
Ahh yeah that's throwing an error since you don't have equivalent nodeup versions uploaded to s3. Let me double check the code, then test locally.

Reply:
I was able to successfully test it, so great work!